### ISSUE TITLE
Context: This is the overridden method ('install').

### ISSUE THREAD

**Issue by softmarshmallow** [jonataslaw/get#54]
after adding get, i get this message, from macos build

```

Compiler message:

../../../../../.pub-cache/hosted/pub.dartlang.org/get-1.17.3/lib/src/snackbar/snack_route.dart:279:8: Error: The method 'SnackRoute.install' has more required arguments than those of overridden method 'OverlayRoute.install'.

  void install(OverlayEntry insertionPoint) {

       ^

../../../../../development/flutter/packages/flutter/lib/src/widgets/routes.dart:41:8: Context: This is the overridden method ('install').

  void install() {

       ^
Running pod install...                                              1.5s
../../../../../.pub-cache/hosted/pub.dartlang.org/get-1.17.3/lib/src/routes/default_route.dart:242:9: Error: No named parameter with the name 'animation'.
        animation: animation,
        ^^^^^^^^^
../../../../../development/flutter/packages/flutter/lib/src/cupertino/route.dart:435:3: Context: Found this candidate, but the arguments don't match.
  CupertinoFullscreenDialogTransition({
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dartlang.org/get-1.17.3/lib/src/snackbar/snack_route.dart:289:18: Error: Too many positional arguments: 0 allowed, but 1 found.
Try removing the extra positional arguments.
    super.install(insertionPoint);
                 ^
Building macOS application...
[INFO:persistent_cache.cc(338)] PersistentCache::UpdateAssetPath: /Users/softmarshmallow/Documents/Apps/inked/inked-flutter/inked/build/macos/Build/Products/Debug/inked.app/Contents/Frameworks/App.framework/Resources/flutter_assets
flutter: started application..
```

**Comment by jonataslaw** [comment]
> after adding get, i get this message, from macos build
> 
> ```
> 
> Compiler message:
> 
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-1.17.3/lib/src/snackbar/snack_route.dart:279:8: Error: The method 'SnackRoute.install' has more required arguments than those of overridden method 'OverlayRoute.install'.
> 
>   void install(OverlayEntry insertionPoint) {
> 
>        ^
> 
> ../../../../../development/flutter/packages/flutter/lib/src/widgets/routes.dart:41:8: Context: This is the overridden method ('install').
> 
>   void install() {
> 
>        ^
> Running pod install...                                              1.5s
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-1.17.3/lib/src/routes/default_route.dart:242:9: Error: No named parameter with the name 'animation'.
>         animation: animation,
>         ^^^^^^^^^
> ../../../../../development/flutter/packages/flutter/lib/src/cupertino/route.dart:435:3: Context: Found this candidate, but the arguments don't match.
>   CupertinoFullscreenDialogTransition({
>   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-1.17.3/lib/src/snackbar/snack_route.dart:289:18: Error: Too many positional arguments: 0 allowed, but 1 found.
> Try removing the extra positional arguments.
>     super.install(insertionPoint);
>                  ^
> Building macOS application...
> [INFO:persistent_cache.cc(338)] PersistentCache::UpdateAssetPath: /Users/softmarshmallow/Documents/Apps/inked/inked-flutter/inked/build/macos/Build/Products/Debug/inked.app/Contents/Frameworks/App.framework/Resources/flutter_assets
> flutter: started application..
> ```

Read and documentation and use the correct version of Get.

**Comment by softmarshmallow** [comment]
sorry, still getting the same error with `get: ^2.0.1-dev` with flutter `Flutter (Channel dev, v1.18.0-8.0.pre, on Mac OS X 10.15.1 19B88, locale en-US)`
```

Compiler message:
../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/root/root_widget.dart:82:23: Error: Type 'ActionFactory' not found.
  final Map<LocalKey, ActionFactory> actions;
                      ^^^^^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/snackbar/snack_route.dart:279:8: Error: The method 'SnackRoute.install' has more required arguments than those of overridden method 'OverlayRoute.install'.
  void install(OverlayEntry insertionPoint) {
       ^
../../../../../development/flutter/packages/flutter/lib/src/widgets/routes.dart:41:8: Context: This is the overridden method ('install').
  void install() {
       ^
../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/routes/default_route.dart:242:9: Error: No named parameter with the name 'animation'.
        animation: animation,
        ^^^^^^^^^
../../../../../development/flutter/packages/flutter/lib/src/cupertino/route.dart:435:3: Context: Found this candidate, but the arguments don't match.
  CupertinoFullscreenDialogTransition({
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/snackbar/snack_route.dart:289:18: Error: Too many positional arguments: 0 allowed, but 1 found.
Try removing the extra positional arguments.
    super.install(insertionPoint);
                 ^
../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/root/root_widget.dart:82:23: Error: 'ActionFactory' isn't a type.
  final Map<LocalKey, ActionFactory> actions;
                      ^^^^^^^^^^^^^
```

**Comment by jonataslaw** [comment]
> sorry, still getting the same error with `get: ^2.0.1-dev` with flutter `Flutter (Channel dev, v1.18.0-8.0.pre, on Mac OS X 10.15.1 19B88, locale en-US)`
> @jonataslaw
> 
> ```
> 
> Compiler message:
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/root/root_widget.dart:82:23: Error: Type 'ActionFactory' not found.
>   final Map<LocalKey, ActionFactory> actions;
>                       ^^^^^^^^^^^^^
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/snackbar/snack_route.dart:279:8: Error: The method 'SnackRoute.install' has more required arguments than those of overridden method 'OverlayRoute.install'.
>   void install(OverlayEntry insertionPoint) {
>        ^
> ../../../../../development/flutter/packages/flutter/lib/src/widgets/routes.dart:41:8: Context: This is the overridden method ('install').
>   void install() {
>        ^
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/routes/default_route.dart:242:9: Error: No named parameter with the name 'animation'.
>         animation: animation,
>         ^^^^^^^^^
> ../../../../../development/flutter/packages/flutter/lib/src/cupertino/route.dart:435:3: Context: Found this candidate, but the arguments don't match.
>   CupertinoFullscreenDialogTransition({
>   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/snackbar/snack_route.dart:289:18: Error: Too many positional arguments: 0 allowed, but 1 found.
> Try removing the extra positional arguments.
>     super.install(insertionPoint);
>                  ^
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1/lib/src/root/root_widget.dart:82:23: Error: 'ActionFactory' isn't a type.
>   final Map<LocalKey, ActionFactory> actions;
>                       ^^^^^^^^^^^^^
> ```
> 
> and after few while, i found out that pubspec.lock's get version did not match and manually changed as 2.0.1-dev.
> 
> ```
>   get:
>     dependency: "direct main"
>     description:
>       name: get
>       url: "https://pub.dartlang.org"
>     source: hosted
>     version: "2.0.1-dev"
> ```
> 
> after that i get this new error
> 
> ```
> Compiler message:
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1-dev/lib/src/root/root_widget.dart:82:23: Error: Type 'ActionFactory' not found.
>   final Map<LocalKey, ActionFactory> actions;
>                       ^^^^^^^^^^^^^
> Building macOS application...
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1-dev/lib/src/root/root_widget.dart:82:23: Error: 'ActionFactory' isn't a type.
>   final Map<LocalKey, ActionFactory> actions;
>                       ^^^^^^^^^^^^^
> ../../../../../.pub-cache/hosted/pub.dartlang.org/get-2.0.1-dev/lib/src/root/root_widget.dart:199:23: Error: The argument type 'Map<LocalKey, invalid-type>' can't be assigned to the parameter type 'Map<Type, Action<Intent>>'.
>  - 'Map' is from 'dart:core'.
>  - 'LocalKey' is from 'package:flutter/src/foundation/key.dart' ('../../../../../development/flutter/packages/flutter/lib/src/foundation/key.dart').
>  - 'Type' is from 'dart:core'.
>  - 'Action' is from 'package:flutter/src/widgets/actions.dart' ('../../../../../development/flutter/packages/flutter/lib/src/widgets/actions.dart').
>  - 'Intent' is from 'package:flutter/src/widgets/actions.dart' ('../../../../../development/flutter/packages/flutter/lib/src/widgets/actions.dart').
>       actions: widget.actions,
>                       ^
> ```

In fact, the version released yesterday of the Dev branch has problems with two internal packages that used the same variable, and is causing conflict in the lib. I'm going to fix the mistake for myself now.
Thanks for mentioning the version you are using, it was trivial to track down the error.

**Comment by jonataslaw** [comment]
Fixed on 2.0.2-dev

**Comment by softmarshmallow** [comment]
@jonataslaw thank you.
